raster3d/r3.out.vtk/errorHandling.c

```diff
@@ -62,7 +62,7 @@ int close_input_raster3d_map(void *map)
 {
     if (map != NULL) {
 	if (!Rast3d_close(map)) {
-	    G_warning(_("Unable to close 3D raster map <%s>"), map);
+	    G_warning(_("Unable to close 3D raster map <%p>"), map);
```

Comment:
All these `void *` variables in the raster3d lib and raster3d modules are a nightmare...

Reply:
suggestion...?

Reply:
Out of the scope of this PR. Please create a new issue that these `void *` variables in [r3.out.vtk](https://github.com/OSGeo/grass/blob/master/raster3d/r3.out.vtk/globalDefs.h) need to be changed.

Reply:
See PR #1291 for possible solution.